keps/sig-node/4800-cpumanager-split-l3cache/README.md

```diff
@@ -0,0 +1,1161 @@
+
+![Figure to the motivation](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/l3-alignment-motivation.png)
+
+The figure below highlights performance gain when pod placement is aligned to an L3 group. In this exampleTPCC/My-SQL  is the workload deployed. There was a 18% uplift compared to default behavior.  Other workloads might see  higher gains.  For Stream workload, highlighted in the original design doc, the performance uplift was approximately 20%.
+
+![Figure to the results of l3 cache alignment](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/results.png) ![Figure of results](https://github.com/ajcaldelas/enhancements/blob/c7e423439608001cc837e2f5629dc87935353611/keps/sig-node/4800-cpumanager-split-l3cache/images/chart.png)
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+- Introduce a new CPU Manager policy option that assigns CPU within the same grouping of uncorecache or L3 cache to pods and container scope.  
+- Minimize the number of CPUs assignments from different uncorecache/L3 grouping to pods and container scope.
+- Cross-die could also decrease performance.  Add support for multiple socket systems.
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+This proposal does not aim to modify CPU assignments for CPU Manager policy set to none.  This does not alter the behavior of existing static policy options such as full-pcpus-only.
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+- Add a new static policy option to fine-tune of CPU assignments to align by uncorecache/L3 cache grouping.
+- Topology struct already contains CPUDetails which is a map of CPUInfo.  CPUInfo knows about NUMA, socket, and core IDs associated with a CPU.  We just need to add a new member called uncorecacheId that tracks whether the CPU is part of a split L3. Functionality to support this was merged in cadvisor with these pull-requests [cadvisor/pr-2849](https://github.com/google/cadvisor/pull/2849) and [cadvisor/pr-2847](https://github.com/google/cadvisor/pull/2847/)
```

Comment:
In Kubernetes, there is an interest to drop cadvisor in favor of CRIStats. @haircommander How would CRIStats fit in with CPUInfo?

Reply:
that's machine level information, not container, so it'll continue to be provided by cadvisor